pkg/settings/setting.go

```diff
@@ -236,6 +239,130 @@ var (
 	UIPreferred = NewSetting("ui-preferred", "vue")
 )
 
+func getClusterAgentDefaultAffinity() string {
```

Comment:
Are we going to start using these default affinity values when we start rendering the `cattle-cluster-agent` manifests?

Reply:
Can we move this stuff out into its own file in the same package? In its current state, it makes an already difficult-to-grok file even bigger.

Reply:
I believe so yes. Those default values will be set in the cluster-agent manifest if no override is provided. We could, but as I understand it when we do the backend plumbing I'll move this default affinity object code out anyway and the settings will pull the default affinity via a middle function.

Reply:
@a-blender , this matches what I brought up in our offline sync last week.
The logic that actually renders this default affinity into YAML manifest will likely need object/YAML representation of it while setting is in JSON. So it makes sense to have a func that returns actual `Affinity` object so it can be used in both places.

But that's not important to address as part of this "data model" PR.